pre_commit_hooks/auto_indent.py

```diff
@@ -0,0 +1,183 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import print_function
+
+import argparse
+import ast
+import io
+import re
+import textwrap
+import traceback
+from operator import itemgetter
+
+
+RETURN_CODE = dict(
```

Comment:
this is of dubious value as a dictionary -- I'd actually remove it entirely or if you really want constants just make them module-level constants